cmd/prysmctl/validator/withdraw.go

```diff
@@ -87,6 +89,13 @@ func callWithdrawalEndpoints(ctx context.Context, host string, request []*apimid
 	if err != nil {
 		return err
 	}
+	fork, err := client.GetFork(ctx, "head")
```

Comment:
Isn't it simpler to check that `CapellaForkEpoch` is set and then that current epoch is higher than that?

Reply:
yeah that's true, at first I thought I somehow I could compare the output of fork response with version.Capella somehow and winded up on this road 😓